__tests__/app.test.js

```diff
@@ -4,6 +4,7 @@ const db = require("../db/connection");
 const seed = require("../db/seeds/seed");
 const testData = require("../db/data/test-data");
 const endpointsData = require('../endpoints.json')
+const format = require('pg-format');
```

Comment:
this import looks like it is unused so should be removed

Reply:
noted and amended